doc/man/scons.xml

```diff
@@ -1117,6 +1117,20 @@ the help message not to be displayed.
   </listitem>
   </varlistentry>
 
+  <varlistentry>
+  <term><option>--hash-format=<replaceable>ALGORITHM</replaceable></option></term>
+  <listitem>
+<para>Set the hashing algorithm used by SCons to
+<replaceable>ALGORITHM</replaceable>.
+This value determines the hashing algorithm used in generating content signatures
+or &f-link-CacheDir; keys.</para>
+
+<para>The supported list of values are: md5, sha1, and sha256.
+However, the Python interpreter used to run SCons must have the corresponding
+support available in hashlib to use the specified algorithm.</para>
+  </listitem>
+  </varlistentry>
+
```

Comment:
Add a note here that it changes the sconsign name.

Reply:
Done, thanks.